modules/bibcirculation/lib/bibcirculation_dblayer.py

```diff
@@ -2700,6 +2700,11 @@ def get_purchase_request_borrower_details(ill_request_id):
     else:
         return None
 
+def update_ill_request_letter_number(ill_request_id, overdue_letter_number):
+    query = ('UPDATE crcILLREQUEST set overdue_letter_number={0} '
+             'where id="{1}"')
+    run_sql(query.format(overdue_letter_number, ill_request_id))
```

Comment:
This would permit a theoretical SQL injection if `overdue_letter_number` contains an unexpected string with quotes and backslashes, for a reason or another.

Can you please use `run_sql(query, (param1, param2))` technique?  In this way the parameters will be always properly escaped.

- "Supply  your run_sql() arguments in a tuple to protect against SQL injection" 
  https://github.com/inveniosoftware/invenio/blob/maint-1.2/CONTRIBUTING.rst